meta/building.md

```diff
@@ -28,6 +28,18 @@ After building the binutils you can build BRUTAL by running:
 
 `make all TOOLCHAIN=gnu`
 
+## On macOS
+
+If you use [homebrew package manager](https://brew.sh), you can obtain the dependencies by running `brew install qemu nasm xorriso llvm binutils wget`.
```

Comment:
```suggestion
If you use [Homebrew package manager](https://brew.sh), you can obtain the dependencies by running `brew install qemu nasm xorriso llvm binutils wget pkg-config sdl2`.
```

Reply:
@jjuliano this will also install a more recent version of `llvm`. e.g: `13.0.1`. Do we want to pin to `llvm@12`?

Reply:
We target >= llvm 12 and GCC >= 11 anything older doesn't have a good enough C2x support

Reply:
@sleepy-monax ok Roger that!